### DISCUSSION TITLE
Any way to stop dead NAND 4's from panicking in ramdisk?

### DISCUSSION THREAD

**Discussion by Lost-Entrepreneur439** [LukeZGD/Legacy-iOS-Kit#638] [Q&A]
Dead NAND 4's are as common as dirt, and I have something in a ramdisk I want to test on my dead NAND 4, but even in a ramdisk, dead NAND 4's will still panic after about 10 seconds. Is there any way to stop those panics?

**Comment by LukeZGD** [comment]
Unfortunately I don't think so, dead NAND is really the showstopper here

**Reply by Lost-Entrepreneur439** [reply]
What even causes them to panic? Shouldn't a ramdisk have to make absolutely no NAND access?

**Reply by LukeZGD** [reply]
Not sure tbh.. It's also like how ramdisks cant be booted without a battery from what I observed on some devices

**Reply by Lost-Entrepreneur439** [reply]
> Not sure tbh.. It's also like how ramdisks cant be booted without a battery from what I observed on some devices

could it perhaps be an ios version issue? im not sure what ios version it uses for the ramdisk but  I wonder if older or newer versions wouldn't have the issue.
